templates/ortools/app.yaml

```diff
@@ -1,4 +1,6 @@
+# This manifest holds the information the app needs to run on the Nextmv Cloud.
 type: python
 runtime: ghcr.io/nextmv-io/runtime/ortools:latest
+# List all files/directories that should be included in the app.
```

Comment:
```suggestion
# List all files/directories that should be included in the app. Globbing (e.g.: configs/*.json) is supported.
```
We could remark it here, but it probably fits the docs more and is verbose here.

Reply:
Listo.